database/usage_tracker.py

```
@@ -318,36 +256,106 @@ def track_chat_completions_create(self, model: str, operation_type: str, **kwarg
+                        COUNT(*) as calls,
+                        SUM(total_tokens) as tokens,
+                        SUM(estimated_cost) as cost
+                    FROM api_usage_tracking 
+                    WHERE session_id = ? AND project_name = ?
+                    GROUP BY agent_type
+                    ORDER BY cost DESC
+                ''', (self.session_id, self.current_project))
+                
+                agent_breakdown = cursor.fetchall()
+                if agent_breakdown:
+                    print(f"   Agent Breakdown:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)